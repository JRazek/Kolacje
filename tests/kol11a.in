100 80
70 34 4 24 18 66 67 58 45 17 55 51 63 63 34 62 68 56 16 30 52 8 31 60 67 12 16 67 68 63 36 2 39 3 31 5 42 39 10 72 41 79 69 42 51 5 54 77 68 50 32 32 75 42 51 61 59 42 20 12 39 45 7 27 26 61 11 58 79 28 12 60 30 75 49 12 32 11 80 76 19 67 20 8 33 4 63 29 58 9 25 32 4 51 35 72 29 46 71 45
96 7 254
15 14 823
55 61 984
26 67 127
51 1 164
87 7 16
80 33 741
30 36 916
65 88 472
93 76 748
77 98 155
82 81 268
31 72 242
4 64 190
16 15 54
93 17 218
17 52 616
90 49 134
46 72 308
98 13 404
84 58 899
74 24 297
21 62 481
97 47 675
87 28 751
56 85 326
33 35 56
67 95 110
12 41 419
86 71 873
43 2 777
27 74 908
37 95 538
36 91 556
3 91 104
45 16 217
51 63 574
70 53 214
6 50 803
97 28 381
37 94 274
100 29 725
29 9 110
60 88 122
20 11 178
40 66 226
60 39 0
23 99 946
78 85 664
48 12 698
30 38 634
44 3 923
35 14 319
42 19 353
63 92 882
83 13 467
43 56 172
73 75 395
8 89 842
20 92 795
5 18 438
45 34 744
70 94 453
89 53 243
32 99 720
2 47 851
26 79 47
4 69 477
24 46 83
23 48 880
100 54 64
68 44 23
78 59 160
9 62 605
84 57 188
90 38 459
73 31 583
61 40 565
52 86 394
77 79 960
11 76 498
69 1 289
34 32 303
58 22 396
57 10 150
19 64 57
25 6 632
65 42 828
59 5 969
71 8 176
80 54 68
81 27 57
68 96 911
39 41 34
50 49 428
22 83 690
55 82 248
18 75 39
21 25 599
100
97 76 7
27 93 52
38 49 57
75 52 38
8 72 30
59 35 24
45 48 3
58 66 71
8 57 72
99 7 36
4 20 18
57 79 37
72 5 78
17 75 6
65 91 44
28 3 74
20 45 8
21 98 38
20 66 67
40 77 26
17 62 53
54 2 41
97 29 64
8 6 41
59 89 40
84 32 23
93 21 34
2 55 29
60 67 40
24 18 29
59 68 78
80 78 56
40 95 58
33 99 40
84 60 69
82 38 72
33 13 31
41 14 8
37 17 34
68 51 31
38 80 44
100 18 80
89 56 3
74 5 55
73 91 25
91 31 53
18 96 9
11 30 23
84 70 67
85 73 11
52 10 19
85 37 50
50 4 77
60 24 51
27 30 12
78 100 12
75 6 61
10 51 63
77 77 5
37 50 7
1 56 54
3 96 35
8 22 79
7 62 78
65 10 80
24 9 31
4 79 16
41 26 73
36 52 25
20 4 13
69 73 8
20 41 52
97 97 10
50 14 44
55 69 54
11 90 20
9 46 9
15 94 15
76 29 71
99 3 20
78 35 33
74 2 69
27 69 4
68 22 32
20 30 46
19 70 55
95 24 78
36 45 11
97 55 37
47 48 35
33 39 27
4 78 59
58 33 79
46 88 67
60 52 69
95 96 71
12 4 47
93 32 76
99 35 74
85 18 47
