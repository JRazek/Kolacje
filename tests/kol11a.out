23410
32045
-1
-1
32726
31357
3593
42590
57073
27800
51475
-1
-1
-1
-1
-1
28397
-1
32969
40252
-1
26011
-1
19142
-1
-1
15634
9986
-1
11714
-1
17176
39008
-1
55391
13681
19484
34728
27805
18170
-1
53153
29752
57397
8829
-1
15486
-1
45138
2665
74472
31262
13063
24213
38114
28082
12364
13109
52648
21971
22794
53337
18334
-1
15467
19820
10224
-1
22185
-1
49354
23183
35062
-1
28490
33698
16209
-1
13690
25063
14839
5453
27917
29646
33227
8428
-1
23636
-1
42293
9877
50796
21037
24252
47129
27597
-1
16838
-1
-1
